crates/wasm-smith/src/config.rs

```diff
@@ -253,13 +253,24 @@ define_config! {
         /// specification](https://webassembly.github.io/spec/core/syntax/instructions.html);
         /// e.g., numeric, vector, control, memory, etc.
         ///
+        /// Additionally, we include finer-grained categories which exclude floating point
+        /// instructions, e.g. [`InstructionKind::NumericNonFloat`] is a subset of
+        /// [`InstructionKind::Numeric`] consisting of all numeric instructions which
+        /// don't involve floats.
+        ///
         /// Note that modifying this setting is separate from the proposal
         /// flags; that is, if `simd_enabled() == true` but
         /// `allowed_instruction()` does not include vector instructions, the
         /// generated programs will not include these instructions but could
         /// contain vector types.
         pub allowed_instructions: InstructionKinds = InstructionKinds::all(),
 
+        /// Indicates whether we should abstain from generating floating point
+        /// instructions and types.
+        ///
+        /// Defaults to `false`.
+        pub disallow_floats: bool = false,
```

Comment:
Would you be ok inverting this to `enable_floats`? (or `allow_floats` or similar). I find "positive" config features are generally easier to understand and rationalize how they compose rather than "negative" features. A minor thing though so doesn't matter too much either way

Reply:
Sure, I switched it to `allow_floats`